src/common/utility.js

```diff
@@ -129,8 +129,10 @@ const dmt5_android_url =
 const dmt5_ios_url = 'https://download.mql5.com/cdn/mobile/mt5/ios?server=Deriv-Demo,Deriv-Server'
 const dp2p_google_play_url =
     'https://play.google.com/store/apps/details?id=com.deriv.dp2p&hl=en&gl=US'
-const cfd_warning_height_desktop = 8
-const cfd_warning_height_tablet = 12
+const CFDWarningHeight = {
```

Comment:
we should use `snake_case` for variables

Reply:
shouldn't we treat objects the same as a class?

Reply:
It's used as a variable in this context, so it's better to be treated as a variable to avoid any confusion.

Reply:
okay. done.